modules/macs/guest/apply.sh

```diff
@@ -85,14 +85,14 @@ EOF
     ls -la /private/var/run || true
     ln -s /private/var/run /run || true
     nix-channel --add https://nixos.org/channels/nixos-19.03 nixpkgs
-    nix-channel --add https://github.com/LnL7/nix-darwin/archive/master.tar.gz darwin
+    nix-channel --add https://github.com/LnL7/nix-darwin/archive/1d09aa2681bb3fbebb6cf9a40c21c6c871117146.tar.gz darwin # TODO, fix this
```

Comment:
`substituteAll` is ran over this file, so you can do `@rev@` and then nix can replace it with a value from the niv data

Reply:
Substituted at the url level since the full url (account, repo, rev) may change with the switching of a fork.